src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -8183,6 +8183,11 @@ void Unit::AttackedTarget(Unit* target, bool canInitialAggro)
     if (Unit* targetOwner = target->GetCharmerOrOwner())
         targetOwner->EngageWithTarget(this);
 
+    //Patch 3.0.8: All player spells which cause a creature to become aggressive to you will now also immediately cause the creature to be tapped.
+    if (Creature* creature = target->ToCreature())
+        if (!creature->hasLootRecipient() && GetTypeId() == TYPEID_PLAYER)
```

Comment:
what about pets (pet taunt)

Reply:
Pets are not tagging based on damage atm, should they?
I didn't play pet classes on retail.

Reply:
Any relation to the case when pets do 100% damage, hence no loot for the player?

Reply:
no, pets don't tag at all right now.

Reply:
anyway, this PR doesn't touch pets, and i'm not going to either until some documentation shows up.